### ISSUE TITLE
Compiling errors of OFP

### ISSUE THREAD

**Issue by bestgen** [OpenFastPath/ofp#201]
Hello. My development environment is CentOS7 and have manually installed ODP before.
Then I download ofp codes and try to compile OFP. 
I type in `./configure --with-libconfig=/usr/local/lib --with-odp=/usr/local/bin`, then the terminal shows：

```
checking libconfig.h presence... yes
checking for libconfig.h... yes
checking for library containing config_init... no
configure: error: Unable to find libconfig library
```
While I have successfully installed libconfig and use it to compile ODP.
```
[admin@localhost ofp]$ rpm -qa| grep libconfig
libconfig-1.4.9-5.el7.x86_64
```
And there is config_init funciton in libconfig.h.
`extern LIBCONFIG_API void config_init(config_t *config);`
Does anyone have some ideas ?  I am a noob. Thx.

**Comment by RadulescuValentin** [comment]
Hi,

Did you try to install libconfig-dev.
https://github.com/OpenFastPath/ofp/blob/master/README.md specify you need to install libconfig-dev.

**Comment by bestgen** [comment]
Hello Radulescu and thanks for your reply. I have installed the development package before but it still doesn't work.  
```
[admin@localhost ofp]$ rpm -qa|grep libconfig-devel
libconfig-devel-1.4.9-5.el7.x86_64
```
I don't know whether next part matters to this problem or not but I write down just in case it does.

I downloaded ofp package before I got this problem.  However, when I try to compile the codes, the terminal shows errors and says cannot find gcc to compile it (which is odd). So I followed the procedures in the README file, using git to get another ofp file and then it can be configured at least.
